core/src/main/java/com/novoda/noplayer/exoplayer/InternalErrorListener.java

```diff
@@ -1,33 +1,19 @@
 package com.novoda.noplayer.exoplayer;
 
-import android.media.MediaCodec;
-
-import com.google.android.exoplayer.MediaCodecTrackRenderer;
-import com.google.android.exoplayer.audio.AudioTrack;
+import com.google.android.exoplayer2.ExoPlaybackException;
 
 import java.io.IOException;
 
 /**
- * A listener for internal errors.
- *
+ * TODO: Add additional logging when developing DRM. A listener for internal errors.
```

Comment:
Is `A listener for internal errors.` related to the TODO? Doesn't seem so. Also why is the TODO in the JavaDoc? ⚠️